lib/screens/weather_screen.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+import '../widget/days_row.dart';
+
+class HomePage extends StatelessWidget{
```

Comment:
add constructor with key. Maybe install flutter snippets in order to build stateless widgets easily